src/pretix/presale/views/user.py

```diff
@@ -0,0 +1,67 @@
+from django.contrib import messages
+from django.shortcuts import redirect
+from django.utils.functional import cached_property
+from django.utils.translation import ugettext_lazy as _
+from django.views.generic import TemplateView
+
+from pretix.base.services.mail import mail
+from pretix.multidomain.urlreverse import build_absolute_uri, eventreverse
+from pretix.presale.forms.checkout import ContactForm
```

Comment:
I don't *really* fancy using that form as it is semantically a very different thing.